app/serializers/winning_bid_serializer.rb

```diff
@@ -0,0 +1,16 @@
+class WinningBidSerializer < ActiveModel::Serializer
+  root false
+
+  attributes(
+    :amount,
+    :bidder_id
+  )
+
+  def amount
+    object.amount
+  end
+
+  def bidder_id
+    object.bidder_id
```

Comment:
Thanks, Chrisrian. We just deployed a bug fix in our latest PR.

On Wednesday, May 4, 2016, Christian G. Warden <notifications@github.com>
wrote:

> In app/serializers/winning_bid_serializer.rb
> <https://github.com/18F/micropurchase/pull/484#discussion_r62115116>:
>
> > @@ -0,0 +1,16 @@
> > +class WinningBidSerializer < ActiveModel::Serializer
> > +  root false
> > +
> > +  attributes(
> > +    :amount,
> > +    :bidder_id
> > +  )
> > +
> > +  def amount
> > +    object.amount
> > +  end
> > +
> > +  def bidder_id
> > +    object.bidder_id
>
> The current bidder is exposed during active auctions. Looks like you need
> to use veiled_bidder.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/18F/micropurchase/pull/484/files/18d0fb8965f8d860a8b7b861ceabda92f4d5fd2b#r62115116>
>